### DISCUSSION TITLE
[Core] Ability to dynamically update flags for a standalone instance

### DISCUSSION THREAD

**Discussion by abhinavsingh** [abhinavsingh/proxy.py#1020] [Ideas]
Motivation here is that developers would like to update `proxy.py` instance config on the fly.  While this is possible in the embedded mode `(programmatically)`, but not for standalone instance `(daemon)` mode.

**Comment by Wikidepia** [comment]
How to dynamically update flags programmatically? I can't seem to find it.

Thank you!

**Reply by abhinavsingh** [reply]
Which flag do you wish to override at runtime?

**Reply by Wikidepia** [reply]
I would like to update proxy_pool flags
